.github/build.in.yml

```diff
@@ -63,6 +63,18 @@ jobs:
         run: |
           lake exe lint-style
 
+      - name: "check the formatting of the PR title and for missing or inconsistent labels"
+        if: github.event.pull_request.draft == false && github.event.pull_request.base == 'master'
+        shell: bash
+        env:
+          TITLE: ${{ github.event.pull_request.title }}
+        run: |
+          set -eo pipefail
+          echo "$TITLE"
+          label_names=$(echo "${{ toJson(github.event.pull_request.labels) }}" | jq -r '.[].name')
+          echo "$label_names"
+          lake exe check-title-labels "$TITLE" "$label_names"
+
```

Comment:
do we want to have this run every time a commit gets pushed? or maybe only on creating the pr and on editing the pr title?